lib/StaticNarrative/exporter/app_processor.py

```diff
@@ -72,47 +78,17 @@ def _process_app_params(
         """
         info = {"input": [], "output": [], "parameter": []}
 
-        # two passes
-        # 1. Make a lookup table for UPA -> object info
-        upas = {}
-        for p in spec_params:
-            upas.update(self._make_upa_dict(param_values.get(p["id"]), p))
```

Comment:
we already have this data (all will be revealed when we get to `exporter.py`) so no need for this step